README.md

```diff
@@ -29,6 +29,40 @@ Tutorials in the **Intro-Full** directory cover
 
 All the tutorial folders can be built using either the `Makefile` or the CMake `CMakeLists.txt` file in each folder.
 
+## CMake
+
+CMake can build against an installed Kokkos library or download one automatically using `FetchContent`.
+
+Without any Kokkos already installed in an exercise directory, one can run the following:
+
+```shell
+cmake -B build_dir . # -DKokkos_* options
+cmake --build build_dir
+```
+
+Kokkos options are described in [CMake options](https://kokkos.org/kokkos-core-wiki/keywords.html).
+
+For example, OpenMP CPU exercises can be built as:
+```shell
+cmake -B build_openmp -S . -DKokkos_ARCH_NATIVE=ON -DKokkos_ENABLE_OPENMP=ON
+cmake --build build_openmp --parallel
+```
+
+On Mac, if OpenMP is not available, one can use the Threads backend:
+```shell
+cmake -B build_threads -S . -DKokkos_ARCH_NATIVE=ON -DKokkos_ENABLE_THREADS=ON
+cmake --build build_threads --parallel
+```
+
+For a NVIDIA V100 gpu:
+
+```shell
+cmake -B build_cuda -S . -DKokkos_ARCH_NATIVE=ON Kokkos_ARCH_VOLTA70=ON -DKokkos_ENABLE_CUDA=ON
+cmake --build build_cuda --parallel
+```
+
+To pass an already installed Kokkos library, you can use classical CMake variables, such as `KOKKOS_ROOT` (for newer CMake), `Kokkos_ROOT`, or `CMAKE_PREFIX_PATH`.
```

Comment:
I didn't even know about the all uppercase version.  The doc says it was added in CMake 3.27
I would omit it for simplicity and just say `Kokkos_ROOT` because it will always work and take precedence over the other ones.